tests/test-team/template-mgmt-component-tests/template-mgmt-copy-page.component.ts

```diff
@@ -0,0 +1,157 @@
+import { test, expect } from '@playwright/test';
+import { TemplateMgmtCopyPage } from '../pages/template-mgmt-copy-page';
+import {
+  assertFooterLinks,
+  assertGoBackLink,
+  assertLoginLink,
+  assertNotifyBannerLink,
+  assertSkipToMainContent,
+} from './template-mgmt-common.steps';
+import { TemplateStorageHelper } from '../helpers/template-storage-helper';
+import { TemplateFactory } from '../helpers/template-factory';
+import { TemplateType, templateTypeDisplayMappings } from '../helpers/types';
```

Comment:
Ah this is what I meant. In this comment...

https://github.com/NHSDigital/nhs-notify-web-template-management/pull/225/files#r1888177920